src/pages/stacks-blockchain/technical-specs.md

```diff
@@ -0,0 +1,52 @@
+
+## Proof of Transfer Mining
+
+- Coinbase reward schedule:
+  - 1000 STX/block for first 4 years
+  - 500 STX/block for following 4 years
+  - 250 STX/block for subsequent 4 years
+  - 125 STX/block in perpetuity after that
+- Reward maturity window: 100 blocks, meaning leaders will earn the coinbase reward 100 blocks after the block they successfully mine.
+- Block interval: Stacks blockchain produces blocks at the same rate as the underlying burnchain. For Bitcoin, this is approximately every 10 minutes.
+
+## Stacking
+
+- Stacking works in 2 phases
+  1. Prepare phase: In this phase an "anchor block" is chosen. The qualifying set of addresses ("reward set") is determined based on the snapshot of the chain at the anchor block. Length of prepare phase is 250 blocks (overlaps with reward cycle of the previous phase).
+  2. Reward phase: In this phase miner BTC commitments are distributed amongst the reward set. Reward cycle length is 2000 BTC blocks (~2 weeks).
+- Two reward addresses / block, for a total of 4000 addresses every reward cycle. The addresses are chosen using a VRF (verifiable random function), so each node can deterministically arrive at the same reward addresses for a given block.
+- Stacking threshold: 0.025% of the participating amount of STX when participation is between 25% and 100% and when participation is below 25%, the threshold level is always 0.00625 of the liquid supply of STX.
+- Delegation: An STX address can designate another address to participate in Stacking on its behalf.
+- Pooling: STX holders that individually do not meet the Stacking threshold can pool together their holdings to participate in Stacking. To do this, STX holders must set the (optional) reward address to the "delegate address". For more details, see [this reference](https://docs.blockstack.org/references/stacking-contract#delegate-stx).
+
+## Accounts and Addresses
+
+- Transactions in the Stacks blockchain originate from, are paid for by, and execute under the authority of accounts
+- An account is fully specified by its address + nonce + assets
+- Address contains 2 or 3 fields: 1 byte version, 20 byte public key hash (RIPEMD160(SHA256(input))), optional name (variable length, max 128 bytes)
+- Two types of accounts: standard accounts are owned by one or more private keys; contract accounts are materialized when a smart-contract is instantiated (specified by the optional name field above)
+- Nonce counts number of times an account has authorized a transaction. Starts at 0, valid authorization must include the _next_ nonce value.
+- Assets are a map of all asset types -- STX, any on-chain assets specified by a Clarity contract (e.g. NFTs) -- to quantities owned by that account.
+- Accounts need not be explicit "created" or registered; all accounts implicitly exist and are instantiated on first-use.
+
+## Transactions
+
```

Comment:
as with the above, considering linking to: https://github.com/blockstack/docs/blob/master/src/pages/stacks-blockchain/transactions.md